test/e2e/json-rpc/wallet_requestPermissions.spec.ts

```diff
@@ -36,7 +37,17 @@ describe('wallet_requestPermissions', function () {
         const getPermissions = await driver.executeScript(
           `return window.ethereum.request(${getPermissionsRequest})`,
         );
-        assert.strictEqual(getPermissions[1].parentCapability, 'eth_accounts');
+
+        const grantedPermissionNames = getPermissions
+          .map(
+            (permission: PermissionConstraint) => permission.parentCapability,
+          )
+          .sort();
+
+        assert.deepStrictEqual(grantedPermissionNames, [
+          'endowment:permitted-chains',
```

Comment:
Recording something we discussed on a call: We should verify whether the order has changed here

Reply:
The ordering seems not to matter, additionally I already see `wallet_snaps` as the first item in the wallet_getPermissions result, so any assumption about eth_accounts needing to be first are already invalidated by that. This is good